changelog/no_default_lib.dd

```diff
@@ -0,0 +1,3 @@
+Phobos/DRuntime library dependencies are no longer added when -defaultlib= is given
+
+Phobos/DRuntime library dependencies (i.e. "-lrt", "-ldl", "kernel32") will no longer be added to the linker command when those libraries are disabled with "-defaultlib=".
```

Comment:
Could you please expand this a bit to make it clear that this is a breaking change, which use cases it affects, and what action is required for those cases.

Reply:
Done